pkgs/applications/audio/distrho/default.nix

```diff
@@ -34,31 +31,34 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ pkg-config meson ninja ];
 
-  buildInputs = [
+  buildInputs = rpathLibs ++ [
     alsaLib
-    curl
-    fftwFloat
     freetype
-    glib
     libGL
     libX11
     libXcursor
     libXext
-    libXinerama
-    libXrandr
     libXrender
-    libgcc
-    libglvnd
-    libsecret
   ];
 
+  postFixup = ''
+    for file in \
+      $out/lib/lv2/vitalium.lv2/vitalium.so \
+      $out/lib/vst/vitalium.so \
+      $out/lib/vst3/vitalium.vst3/Contents/x86_64-linux/vitalium.so
+    do
+      patchelf --set-rpath "${lib.makeLibraryPath rpathLibs}:$(patchelf --print-rpath $file)" $file
+    done
+  '';
+
   meta = with lib; {
     homepage = "http://distrho.sourceforge.net/ports";
     description = "Linux audio plugins and LV2 ports";
     longDescription = ''
       Includes:
         arctican-function
         arctican-pilgrim
+        CHOW
```

Comment:
```suggestion
```
https://github.com/DISTRHO/DISTRHO-Ports/commit/cb80ec030896b08471aaa0f2a43fde4b505dbef3

Reply:
I had missed this, thank you.